package.json

```
@@ -6,15 +6,23 @@
   "scripts": {
     "prepublish": "./node_modules/.bin/coffee -c lib.coffee"
```

I think this needs changing to
```suggestion
    "prepare": "./node_modules/.bin/coffee -c lib.coffee"
```
in order to work with `lib.js` removed, since I don't think `prepublish` runs for npm 6

---

Ok, looking at https://github.com/balena-io-modules/zip-part-stream/pull/8 it seems like it's not required but it's still the better option

---

Updated